Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -180,6 +180,13 @@ public void Process(ITelemetry telemetry)
 
         private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry requestTelemetry)
         {
+#if NET45
+            if (requestTelemetry.Url == null && System.Web.HttpContext.Current != null)
```

Comment:
will this method be called sync or async? I thought there is a separate thread for this conversion

Reply:
It seems to be run in sync with all TelemetryProcessors.Process. @tokaplan , could you please confirm?

Reply:
@Mikhail-Pranovich confirmed, this conversion is done sequentially, there is no separate thread.